src/index.ts

```
@@ -2008,6 +2047,94 @@
+				depositDate = null
+			}
+		} else {
+			config.verbose && console.log('No timestamp found in deposit for version', params.contractVersion)
+		}
+	}
+
+	let tokenAmount = '0'
+	let tokenDecimals = null
+	let symbol = null
+	let name = null
+	let tokenURI = null
```

ESLint

[error] 'tokenURI' is never reassigned. Use 'const' instead.

(prefer-const)